### DISCUSSION TITLE
Per user collections

### DISCUSSION THREAD

**Discussion by YoussefAFdez** [gotson/komga#1490] [Ideas]
Hi,

I was wondering why the ability to create or edit collections was reserved for admins only and system wide. I was planning on sharing my konga library with some friends and families, and would like for them to be able to create their own collection so they can sort themselves better. Pretty much like creating a playlist. I will have to give admin privileges to everyone as things are right now.

Would it be much trouble to add the ability for users to create their own libraries linked to their profiles? Or lists of some sorts, maybe I missed up something. Thanks!

**Comment by Ergiler** [comment]
Same issue here.

I share with friend in another country and while I would be ok giving him ability to make collections and reading lists I don’t really want ability to delete everything on server.
More accounts with access to that means more ways fir someone to get access and make mess.

Allowing either access to only those or making lists locked to users would be nice.

Best would be if I could allow others to make new lists and change their own but not touch lists from other users

**Reply by Ergiler** [reply]
While I own he server and add stuff my friend has more knowledge about comics.
If there was some manager role or something that would enable some admin features or if users could do stuff and then admin would have to approve.

**Comment by gotson** [comment]
Superseded by https://github.com/gotson/komga/discussions/2197
